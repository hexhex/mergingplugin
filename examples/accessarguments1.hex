arguments(Pred, Arg) :- &hexfile["hexprog1.hex",""](Answer), &answersets[Answer](AnswerSet), &tuples[Answer, AnswerSet](Pred, Arity), &arguments[Answer, AnswerSet, Pred](RunningNr, 0, Arg).
