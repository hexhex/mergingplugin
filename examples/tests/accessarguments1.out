{arguments(predicateA,y), arguments(predicateB,y)}
